new-frontiers-14/frontier-station-14
# Hammer (Update): Directional fans + tweaks

## About the PR
It's Hammer time.

* Directional fans! #1980
* A proper distro mixer, instead of an air canister. The Hammer was previously exempt due to having O2 and N2 canisters in the small workshop, but I felt it was reasonable to make it consistent with other ships.
* Mapped a box of replacement lights next to the components, because why not. :)
* Lowered the price to $38,000, ~15% markup. The previous price was a 21% markup, a little bit steep for this solo ship with limited moneymaking capability.

## Why / Balance
Compliance. Directional fans are the best, man.

## How to test
1. Buy a Hammer.
2. Try not to think of every problem as a nail.
3. Ensure air is breathable and contained.
4. Check mixer settings to make sure I haven't messed up.

## Media
![image](https://github.com/user-attachments/assets/770c1e1e-1b15-4d64-91bd-44e0de5ae87e)
![image](https://github.com/user-attachments/assets/a4fa14e7-e2f9-425b-a900-c3b79ebe4756)

## Requirements

## Breaking changes
Noperoonies.

**Changelog**
:cl:
- tweak: The SBB Hammer now has directional fans.
